mantidproject/mantid
# Fix Muon Results Table Columns Not Updating

**Description of work.**
Updated column headers in table model

**To test:**
1. Open Muon Analysis
2. load some data
3. Do a fit
4. Go to results table add run_number log and output results table
5. Open the results table produce
6. With this table still open, change the sample log to start time
7. Create a new results table with the same name
8. The previous table should be closed
9. The new table should be updated

Also try running this script:
```
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

tab = CreateEmptyTableWorkspace()
tab.addColumn('double', 'norm')
tab.addColumn('str', 'name')
tab.addColumn('str', 'method')

tab.addRow([1.,"Run;;Group;;a;;Asym;;#1","Estimate"])
tab.addRow([2.,"Run;;Group;;b;;Asym;;#1","Estimate"])
tab.addRow([3.,"Run;;Group;;c;;Asym;;#1","Estimate"])
tab.addRow([4.,"Run;;Group;;d;;Asym;;#1","Estimate"])

# tab.addRow([5.,"Run;;Group;;d;;Asym;;#1","Estimate"])
# tab.addColumn('str', 'added')
```

once you have run this script once, open the table tab and then uncomment the last two lines, Then with the table still open run the last two lines

Fixes #31257 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
